shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -501,6 +506,14 @@ - (void)keyUp:(NSEvent*)event {
   }
 }
 
+- (void)flagsChanged:(NSEvent*)event {
+  if ((event.modifierFlags & NSEventModifierFlagDeviceIndependentFlagsMask) == 0) {
```

Comment:
Does this so the right thing if you, say, press and hold shift, press command, then release command? It seems like the last will be reported as a down rather than an up.

Reply:
No, thanks for catching that. Changed it to keep track of the pressed keys and compare with the new value.